eng/source-build-release.yml

```diff
@@ -59,6 +70,19 @@ stages:
         fi
       displayName: Ensure tag exists
 
+    - script: |
+        set -euxo pipefail
+        cd $(Build.SourcesDirectory)
+        git checkout $(SetTag.Tag)
+        runtime_version=$(cat eng/Version.Details.xml | grep 'Microsoft.NETCore.App.Runtime.win-x64' | grep -P -o -e '(?<=Version=").*?(?=")') # e.g. 6.0.3
+        dotnet_major_minor_version=$(echo "$runtime_version" | head -c 3) # e.g. 6.0
```

Comment:
This probably wont work very well when the major (or minor) versions hit 2 digits.

Maybe try:

```bash
readarray -d '.' -t version_parts <<< $runtime_version  # split version on dots
dotnet_major_minor_version=$(echo "${version_parts[0]}.${version_parts[1]}")
```

Reply:
Nice catch, thanks for the suggestion!